exts/cesium.omniverse/mdl/cesium.mdl

```diff
@@ -220,6 +244,173 @@ float4 compute_base_color(
     return base_color;
 }
 
+// Copied from gltf/pbr.mdl since it's not exported
```

Comment:
I've done very similar things. What happens if `gltf/pbr.mdl` is updated? (I never found a good solution to this question.)

Reply:
I don't have a good solution either. We're absorbing so much of `gltf/pbr.mdl` at this point that I wouldn't be surprised if we copy the whole file.